pkg/trino/datasource.go

```diff
@@ -79,11 +80,20 @@ func (s *TrinoDatasource) Converters() (sc []sqlutil.Converter) {
 }
 
 func (s *TrinoDatasource) SetQueryArgs(ctx context.Context, headers http.Header) []interface{} {
-	user := ctx.Value("X-Trino-User")
-	if user == nil {
-		return []interface{}{}
+	var args []interface{}
+
+	user := ctx.Value(constants.TrinoUserHeader)
+	accessToken := ctx.Value(constants.AccessTokenKey)
```

Comment:
This is in the same package now.

Reply:
Yes, something went wrong with the squash.
fixed.